const/dispatcher.go

```diff
@@ -0,0 +1,6 @@
+package _const
+
+const(
+	DISPATCHGID ="DISPATCH"
+	DISPATCHIP="127.0.0.1"
```

Comment:
I guess this is going to be changed in the future? Probably dispatcher IP should come from the config file.